AK/JsonObject.h

```diff
@@ -136,9 +137,36 @@ template<typename Builder>
 inline void JsonValue::serialize(Builder& builder) const
 {
     switch (m_type) {
-    case Type::String:
-        builder.appendf("\"%s\"", m_value.as_string->characters());
-        break;
+    case Type::String: {
+        auto size = m_value.as_string->length();
+        builder.append("\"");
+        for (size_t i = 0; i < size; i++) {
+            char ch = m_value.as_string->characters()[i];
+            switch (ch) {
+            case '\e':
+                builder.append("\\e");
+                break;
```

Comment:
I don't think `\e` is a thing in JSON. You have to use `\u0033` or whatever the proper code.

Reply:
In current files it is used as "\\x1B". I am going to change it with this value.

Reply:
I don't think JSON supports `\x` escapes either.

Reply:
Yes, json.org says only \u is correct. I amde changes in JSONParser to handle \u. Can you review it for me?